content/en/guides/core/registry/model_registry_eol.md

```diff
@@ -53,30 +56,76 @@ Registered models are now referred to as *collections*.
 | `wandb.init.link_model` | Model Registry specific API | Currently only compatible with legacy model registry |
+| Data | Collections, linked artifact versions, version history, aliases, tags, descriptions, automations and permissions | Copied from legacy Model Registry to W&B Registry |
 
 
 ## Preparing for the migration
 
-W&B will migrate registered models (now called collections) and associated artifact versions from the legacy Model Registry to the W&B Registry. This process will be conducted automatically, with no action required from users.
+No action is required from users before the migration. W&B will handle the transition, ensuring that workflows and references are preserved.
 
-### Team visibility to organization visibility
 
-After the migration, your model registry will have organization level visibility. You can restrict who has access to a registry by [assigning roles]({{< relref "./configure_registry.md" >}}). This helps ensure that only specific members have access to specific registries.
+## During the migration
 
-The migration will preserve existing permission boundaries of your current team-level registered models (soon to be called collections) in the legacy W&B Model Registry. Permissions currently defined in the legacy Model Registry will be preserved in the new Registry. This means that collections currently restricted to specific team members will remain protected during and after the migration. 
+W&B will initiate the migration process. The migration will occur during a time window that minimizes disruption to W&B services. There will be a brief pause (estimated around 2-3 hours) on write operations for a specific team's assets. This means that while your team's legacy Model Registry data is being copied to the new W&B Registry, you won't be able to save new versions or make changes in either that specific legacy Model Registry or its new Registry counterpart. Once the copy is complete for your team, write operations will resume normally in the new W&B Registry. The legacy Model Registry will then become read-only. 
 
-### Artifact path continuity
+The following will happen during the migration:
 
-No action is currently required.
+* W&B will migrate registered models (now called *collections*) and associated artifact versions [from the legacy Model Registry to the W&B Registry]({{< relref "#artifact-paths" >}}). 
+* API calls that reference the legacy Model Registry will be [redirected to the new W&B Registry]({{< relref "#legacy-model-registry-apis" >}}). This means that any existing references to model artifacts in your code or workflows will continue to function without requiring updates.
+* Team level visibility of the legacy Model Registry will be [transitioned to organization level visibility]({{< relref "#team-visibility-to-organization-visibility" >}}) in the W&B Registry.
```

Comment:
Actually, the new organization-level registry will have `restricted` visibility, where the team whose model registry is migrated will be added as a member and the team and model registry admins will be added to the new registry as admins.

Reply:
But they can change those settings after the migration is complete

Reply:
Unless you meant to say that we're moving a team's model registry to the organization-level..?